src/parser.js

```diff
@@ -22,13 +22,24 @@ define([
     }
 
     function getAttributes (element) {
-        var attributes = $(element)[0].attributes,
-            attrMap = {};
+        var attributes = _.chain($(element)[0].attributes)
+                .map(function (value) {
+                    return [value.nodeName, value.nodeValue];
+                }) .object().value(),
+            attrKeys = _.keys(attributes);
+
+        _.each(attrKeys, function (key) {
+            if (key.startsWith('vellum:') &&
```

Comment:
This is used for `rawAttributes` so we want to ignore all the vellum attributes unless it's ignore or lock as those are processed elsewhere